node-playwright-chrome/Dockerfile

```diff
@@ -3,22 +3,20 @@ ARG NODE_VERSION=16
 FROM node:${NODE_VERSION}-bullseye-slim
 
 LABEL maintainer="support@apify.com" Description="Base image for Apify actors using Chrome"
+ENV DEBIAN_FRONTEND=noninteractive
 
 # This image was inspired by https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 
-# Disable chrome auto updates, based on https://support.google.com/chrome/a/answer/9052345
-RUN mkdir -p /etc/default && echo 'repo_add_once=false' > /etc/default/google-chrome
```

Comment:
i think this needs to go first, the link says "Create an empty repository **before** installing Chrome browser:"

Reply:
but I am not entirely sure if this is working, IIRC it wasn't solving the problem we had with the chrome updates going through proxies

Reply:
We'll probably want to add these for chromium too, not just chrome